src/view/view.js

```diff
@@ -126,6 +126,14 @@ export default class View {
 		 */
 		this._postFixersInProgress = false;
 
+		/**
+		 * Internal flag to temporary disable rendering. See usage in the editing controller.
+		 *
+		 * @protected
+		 * @member {Boolean} module:engine/view/view~View#_disabledRendering
+		 */
+		this._disabledRendering = false;
```

Comment:
`_renderingDisabled` would be more grammatical. Plus, you have `_renderingInProgress` above already.